pkg/daemon/metrics.go

```diff
@@ -5,11 +5,55 @@ import (
 	"net/http"
 
 	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-// DefaultBindAddress is the port for the metrics listener
-var DefaultBindAddress = ":8797"
+var (
+	// DefaultBindAddress is the port for the metrics listener
+	DefaultBindAddress = ":8797"
+
+	// RebootCounter is a metric incremented when MCD reboots
+	RebootCounter = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mcd_reboots",
+			Help: "A metric incremented when MCD reboots",
+		}, []string{"node"})
```

Comment:
Should this be a counter rather than a gauge? since it only increment upwards, it seems to fit better with a counter if I'm understanding correctly https://prometheus.io/docs/concepts/metric_types/#counter . 

Although, a gauge metric for active reboots could be useful too 🤔